storage/innobase/srv/srv0srv.cc

```diff
@@ -244,7 +244,8 @@ UNIV_INTERN os_event_t	srv_allow_writes_event;
 
 /** copy of innodb_buffer_pool_size */
 ulint	srv_buf_pool_size;
-const ulint	srv_buf_pool_min_size	= 5 * 1024 * 1024;
+/** Minimum consistent with ha_innodb.cc:min_buffer_pool_size() for PAGE_SIZE_MIN */
+ulint	srv_buf_pool_min_size	= 2 * 1024 * 1024;
```

Comment:
Can we avoid this indirection via a different compilation unit, and simply refer to `MYSQL_SYSVAR(buffer_pool_size)` in `ha_innodb.cc`? Similarly, `srv_buf_pool_def_size` looks unnecessary obfuscation to me. We can simply write `MYSQL_SYSVAR_LONGLONG(buffer_pool_size, …, 128LL << 20, …)` in `ha_innodb.cc`.